.github/workflows/prlabeler.yml

```diff
@@ -12,7 +12,10 @@ jobs:
       contents: read
       pull-requests: write
     runs-on: ubuntu-latest
+
     steps:
     - uses: actions/labeler@v4
       with:
-        repo-token: "${{ secrets.GITHUB_TOKEN }}"
\ No newline at end of file
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
+        configuration-path: ".github/labeler.yml"
+        sync-labels: false
```

Comment:
@sbdtu5498 thanks for tackling this problem!

I see there's a bug report for this problem in the GitHub action repository (https://github.com/actions/labeler/issues/112) and there's a workaround here: https://github.com/actions/labeler/issues/112#issuecomment-1136485391

I am tending towards going with `sync-labels: ""` instead, as indicated in the workaround. What do you think?

I see some examples like https://github.com/DataDog/integrations-core/pull/13015/files

Reply:
Ah! I see. This might be a solution. I can test it and get back on it. I did basic testing on my private repo. "sync-label: false" preserves manually attested labels but also preserves any other labels that were previously added by the action. So if this solution can help us in preserving only manually attested labels, it would be a better fit. @aliok does this thing do the same or does it only preserves manually attested labels?  I can test it if it is not explicitly mentioned but if you have seen anything about it's nature being explicitly mentioned let me know, because I wasn't able to find it out. Thanks!

Reply:
> I did basic testing on my private repo. "sync-label: false" preserves manually attested labels but also preserves any other labels that were previously added by the action.

Ok, I didn't know you did that.

Thanks a lot for the explanation. I guess this approach is better then.

I see the issue I mentioned is going to be fixed in the v5 version of the action. We should use upgrade to that one when it is out there.

Reply:
Yes, it seems to be a good alternative as it is not very heavily required and chances to encounter the given situations are even rarer, so doing more testing would just be a waste of time, When it would be fixed we can integrate it into our workflows.